### ISSUE TITLE
Option to not show time in error logging

### ISSUE THREAD

**Issue by ghost** [cherrypy/cherrypy#1213]
Originally reported by: **Alex Honeywell (Bitbucket: [asoc](http://bitbucket.org/asoc), GitHub: [asoc](http://github.com/asoc))**

---

I'm using the Python logging modules for my application and I have the CherryPy logs also going through my handler.

Since my log format has the time at the start of the string, I found it a minor annoyance that CherryPy puts the time in the error message itself instead of using the LogRecord attributes. 

I added a config option to turn that off, and here's the patch if you think it'd be useful for others.

---
- Bitbucket: https://bitbucket.org/cherrypy/cherrypy/issue/1213

**Comment by bmxp** [comment]
I would like to see that logging in cherrypy would use standard python logging for the internal messages that don't have anything to do with webaccess specific errors.
It would be nice if someone with access to bitbucket.org could provide the patch here on github or otherwise just fix it in core. While this does not have highest importance it would be nice to have a consistent logging.

**Comment by webknjaz** [comment]
@jaraco are the issues still available to you?
@asoc do you still have that patch?
@bmxp if you are able to track it down in the code, feel free to send us a PR :)

**Comment by asoc** [comment]
I don't have the original, but I believe it was essentially just this:

```diff
--- _cplogging.py	2016-12-01 10:32:38.000000000 -0500
+++ _cplogging.py	2016-12-01 10:55:12.000000000 -0500
@@ -161,6 +161,8 @@
         '%(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"'
     )

+    error_with_time = True
+
     logger_root = None
     """The "top-level" logger name.

@@ -216,7 +218,12 @@
         if traceback:
             exc_info = _cperror._exc_info()

-        self.error_log.log(severity, ' '.join((self.time(), context, msg)), exc_info=exc_info)
+        msg = ' '.join((context, msg))
+
+        if self.error_with_time:
+            msg = ' '.join((self.time(), msg))
+
+        self.error_log.log(severity, msg, exc_info=exc_info)

     def __call__(self, *args, **kwargs):
         """An alias for ``error``."""
```

**Comment by adamcparsons** [comment]
I agree, it's annoying to see cherryPys timestamp embedded in the logger message.  This makes it inflexible.  The above patch from asoc logs good but is a little incomplete.  It fails to address the access log as well.  

Perhaps the following would work in addition to asoc:

PY3_notime_format ='{h} {l} {u} "{r}" {s} {b} "{f}" "{a}"'
PY3_time_format ='{h} {l} {u} {t} "{r}" {s} {b} "{f}" "{a}"'
PY2_notime_format = '%(h)s %(l)s %(u)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"'
PY2_time_format = '%(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s"'

access_log_format = PY2_time_format if self.error_with_time else PY2_notime_format
if  six.PY3:
    access_log_format = PY3_time_format if self.error_with_time else PY3_notime_format

**Comment by jaraco** [comment]
> are the issues still available to you?

Yes. I just reviewed the patch, and it does match closely the one repeated above.
